conda-recipes/fairseq-feedstock/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Here, we should have recipe license. You can use the BSD license text from https://github.ibm.com/mldlppc/tracker/wiki/Supplementary-Channel-Wiki. And the package license should go into recipe directory.

Reply:
Thanks @npanpaliya, yes should have read through the document earlier. I have updated the top level license and not added license to recipe directory as it is part of the code that gets checked out and the file is referenced in the `meta.yaml`